Makefile

```diff
@@ -12,88 +12,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The binary to build (just the basename).
-BIN ?= $(wildcard velero-*)
-
-# This repo's root import path (under GOPATH).
 PKG := github.com/vmware-tanzu/velero-plugin-example
+BIN := velero-plugin-example
 
-BUILD_IMAGE ?= golang:1.12-stretch
-
-IMAGE ?= velero/velero-plugin-example
+REGISTRY ?= velero
+IMAGE    ?= $(REGISTRY)/velero-plugin-example
+VERSION  ?= master 
 
-# Which architecture to build - see $(ALL_ARCH) for options.
-# if the 'local' rule is being run, detect the ARCH from 'go env'
+# Which architecture to build.
+# if the 'local' rule is being run, detect the GOOS/GOARCH from 'go env'
 # if it wasn't specified by the caller.
-local : ARCH ?= $(shell go env GOOS)-$(shell go env GOARCH)
-ARCH ?= linux-amd64
-
-platform_temp = $(subst -, ,$(ARCH))
-GOOS = $(word 1, $(platform_temp))
-GOARCH = $(word 2, $(platform_temp))
-
-all: $(addprefix build-, $(BIN))
+local: GOOS ?= $(shell go env GOOS)
```

Comment:
Can we define `GOOS` and `GOARCH` as just variables, outside the `local` target?

Reply:
yeah that should be fine, since they're not being used elsewhere, will do.

Reply:
this also makes me think that i should spend a minute thinking about cross-compilation / targeting other OS/arch combos for the container builds. I know there's a PR up in the AWS repo for this. will see if I need to make any other changes to enable that.

Reply:
After looking at this, our pattern for multi-arch builds is to have a separate Dockerfile per target arch. I don't think there's anything in this PR that would make it difficult to extend to do that (though I'm not going to add it to the example repo since I don't want to over-complicate things).

Reply:
changed the vars.